examples/complete/main.tf

```diff
@@ -25,3 +25,55 @@ module "users_encrypted" {
     Secure = "true"
   }
 }
+
+resource "aws_sqs_queue_policy" "users_unencrypted_policy" {
+  count = var.use_custom_policy ? 1 : 0
```

Comment:
Please remove this `count` and `var.use_custom_policy`.

Examples always self-contained and should be executed without specifying var arguments.

One resource `aws_sqs_queue_policy` is enough in the example.